### ISSUE TITLE
scala-tools.org is gone

### ISSUE THREAD

**Issue by mjwall** [openzipkin/zipkin#368]
With an empty ~/.ivy2 repo and a fresh git clone, I was having trouble getting all the dependencies.  Noticed that on https://github.com/twitter/zipkin/blob/master/project/ZipkinResolver.scala#L29, it was trying to download from scala-tools.org, which appears to be gone.  Replacing

```
"scala-tools.org" at "http://scala-tools.org/repo-releases/",
"testing.scala-tools.org" at "http://scala-tools.org/repo-releases/testing/",
```

with

```
"sonatype-nexus" at "http://scala-tools.org/repo-releases/testing/", //replaced scala-tools.org
```

seems to do the trick.

**Comment by analytically** [comment]
+1

**Comment by nshkrob** [comment]
Can you send a pull request please?

**Comment by mjwall** [comment]
Sure, I'll do a pull request right now.  Thanks

**Comment by mjwall** [comment]
Pull request submitted, #373.  Seems the builds launched by the pull request have been failing.  I don't think this was my pull request.  How does this get resolved?

Thanks

**Comment by sprsquish** [comment]
It's not yours. I pulled in a bad PR and have asked the original author to
help fix it.

On Tue, Apr 15, 2014 at 1:53 PM, Michael Wall notifications@github.comwrote:

> Pull request submitted, #373 https://github.com/twitter/zipkin/pull/373.
> Seems the builds launched by the pull request have been failing. I don't
> think this was my pull request. How does this get resolved?
> 
> Thanks
> 
> ## 
> 
> Reply to this email directly or view it on GitHubhttps://github.com/twitter/zipkin/issues/368#issuecomment-40532894
> .

## 

Jeff Smick
@sprsquish

**Comment by mjwall** [comment]
Closed in #373.  Thanks
